### PR TITLE
[selectors4] Add new tests for :focus-within as universal selector

### PR DESCRIPTION
**Pull request by mrego**
This just adds 2 new simple tests that were useful for my implementation on Blink.

Please @frivoal could you take a look? Thanks!

---
This change is [Reviewable](https://reviewable.io/reviews/w3c/web-platform-tests/5249)

### CODE REVIEW

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/216307709)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 55d304d87eab2c40fae7a87edddc4f0670e8460c
Using browser at version BuildID 20170329100319; SourceStamp 272ce6c2572164f5f6a9fba2a980ba9ccf50770c
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/216307710)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 55d304d87eab2c40fae7a87edddc4f0670e8460c
Using browser at version 59.0.3053.3 dev
Starting 10 test iterations
No tests run.

**Comment by frivoal** [comment]
Looks good to me. Do you want to drop the interact flag first (as in #5370) before I merge?

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5249/)

**Comment by mrego** [comment]
Removed the `interact` flag here too.

File: css/selectors4/focus-within-008.html [hunk 1]
```diff
@@ -1,10 +1,9 @@
 <!DOCTYPE html>
-<meta charset="utf-8">
```
Comment (frivoal) [inline css/selectors4/focus-within-008.html]: Did you remove the `<meta charset="urf-8">` intentionally? That looks like an accident. Revert?

Comment (mrego) [inline css/selectors4/focus-within-008.html]: Not at all, removed the wrong line, sorry.

**Comment by frivoal** [comment]
Looks good, merging.

**Pull request merged**